modules/nw-operator-cr.adoc

```diff
@@ -401,14 +400,23 @@ kind: Network
 metadata:
   name: cluster
 spec:
-  clusterNetwork: 
+  clusterNetwork:
   - cidr: 10.128.0.0/14
     hostPrefix: 23
-  serviceNetwork: 
+  serviceNetwork:
   - 172.30.0.0/16
-  networkType: OVNKubernetes 
-      clusterNetworkMTU: 8900
-    
+  defaultNetwork: <1>
+    type: OVNKubernetes
+    ovnKubernetesConfig:
+      mtu: 1400
+      genevePort: 6081
+      ipsecConfig:
+        mode: Full
+  kubeProxyConfig:
```

Comment:
@bscott-rh Do you want to include KubeProxyConfig in YAML? Are you explaining this at some place?

Reply:
Thanks, I have reverted the code block to its original content. PTAL